### ISSUE TITLE
labels_fine_cityscape test gives only black png files. Is this correct?

### ISSUE THREAD

**Issue by tom-hao** [mcordts/cityscapesScripts#45]
My validation and training sets give images with colored labels, instance labels, and gray scale labels. But when I checked the test set, all three images are completely black. Am I supposed to process these or did my download result in an error?

**Comment by sr425** [comment]
Hi,
sorry I saw the newer post first, for a answer see issue #57, it should refer to the same behaviour.

**Comment by mcordts** [comment]
That's intended. The test folder does not contain any annotations, except for which is known from camera calibration, i.e. the rectification artifacts and the ego vehicle. In order to evaluate on the test set, you need to submit your results to our evaluation server.
